person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Please make sure in the `Person` class a method for adding rentals should be present and make sure the rentals variable is also writable so move it to `attr_accessor`.
  - You can consider the below code

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```